cli/command/stack/remove_test.go

```diff
@@ -72,6 +108,7 @@ func TestRemoveStackSkipEmpty(t *testing.T) {
 	allConfigIDs := buildObjectIDs(allConfigs)
 
 	fakeClient := &fakeClient{
+		version:  "1.30",
```

Comment:
We can probably remove the 1.30 versions once https://github.com/moby/moby/pull/33887 is merged and vendored